zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -175,7 +176,14 @@ private void updateLastExportedPosition() {
   private record CamundaExporterRecordFilter() implements RecordFilter {
     // TODO include other value types to export
     private static final Set<ValueType> VALUE_TYPES_2_EXPORT =
-        Set.of(USER, AUTHORIZATION, DECISION, PROCESS_INSTANCE, VARIABLE, INCIDENT);
+        Set.of(
+            USER,
+            AUTHORIZATION,
+            DECISION,
+            DECISION_REQUIREMENTS,
```

Comment:
This change has nothing to do with this PR. but I've missed adding `DECISION_REQUIREMENTS` to the values to export in the previous one, and I didn't think it needed a separate PR.